README.md

```diff
@@ -30,3 +33,25 @@ to open Orange and check if everything is installed properly.
 
 
 [Anaconda]: https://www.continuum.io/downloads
+
+###From source
+
+To install the add-on from source
+
+    #Clone the repository and move into it
+    git clone https://github.com/biolab/orange3-timeseries.git
+    cd orange3-timeseries
+
+    #Install corresponding wheels for your OS:
+    pip install some-wheel.whl
```

Comment:
What is this supposed to be? :smiley: Most of the time, users copy instructions verbatim.